docs/components/icons/Example.vue

```diff
@@ -0,0 +1,33 @@
+<script setup>
+  import * as icons from '@warp-ds/icons/vue';
+
+  const mappedIconsBySize = Object.keys(icons).reduce((acc, current) => {
+    const matches = current.match(/\d+/g);          
+      
+    if(!acc[ matches[0]]) {
+      acc[matches[0]] = {[current]: icons[current]}
+    } else {
+      acc[matches[0]] = {...acc[matches[0]], ...{[current]: icons[current]}}
+    }
+    return acc;
+      
+  }, {});
+  
+</script>
+
+<template>
+  <section v-for="(size) in Object.keys(mappedIconsBySize)" class="s-bg-primary-subtle rounded-8 p-24 mb-24">
+    <h2>{{size}}</h2>
+    <main class="max-w-screen-xl mx-auto px-32">
+      <div  class="grid gap-24 grid-cols-minmax-100px">
+        <div v-for="(icon, name) in mappedIconsBySize[size]" :key="name">
+          <div class="text-center"><div class="mx-auto mb-8 s-bg rounded-4 h-56 flex items-center justify-center flex-col">
+            <component v-bind:is="icon" class="s-text"></component>
+          </div>
+          <div class="text-12">{{ name }}</div>
```

Comment:
I raised this question in description. I don’t know which is best. Shall I adjust? I had a nice function for that that I can take back

Reply:
I personally would go for just the name, even if it's PascalCase for all framework examples. Just easier to look at icon names when browsing the list imo. Now that I think about it, it might be unnecessary to show the full name of each icon component in the table. I would feel it's enough to see how each component is used in the "Usage" section, meaning that it's always prefixed with `icon`, `w-icon` or `Icon` and postfixed with icon size. But maybe again, it's just me, and others would rather see the full name under each icon 🙂

Reply:
https://github.com/warp-ds/tech-docs/pull/102/commits/b4b596f270abe797a5a304da4aff10b75f3c32b4

Reply:
wdyt about the latest?

<img width="762" alt="image" src="https://github.com/warp-ds/tech-docs/assets/37986637/43efe6ca-2b0c-42e4-8c94-5a6f9bca7596">

Reply:
I like that 😄 I think it's much easier to read.